src/ansys/fluent/core/solver/flobject.py

```diff
@@ -496,6 +497,40 @@ def __eq__(self, other):
             return False
         return self.flproxy == other.flproxy and self.path == other.path
 
+    def get_completer_info(self, prefix="", excluded=None) -> List[List[str]]:
+        """Get completer info of all children.
+
+        Returns
+        -------
+        List[List[str]]
+            Name, type and docstring of all children.
+        """
+        excluded = excluded or []
+        ret = []
+        for k, v in inspect.getmembers(self):
+            if not k.startswith("_") and k not in excluded and k.startswith(prefix):
+                if isinstance(v, Base):
+                    # TODO: Following check hangs for some objects as seen in unittest.
+                    # if v.is_active() and not _is_deprecated(v):
```

Comment:
I'll try this again after https://github.com/ansys/pyfluent/pull/4174 is merged.